test/unit/slotted-items.test.ts

```diff
@@ -0,0 +1,64 @@
+import { LitElement, customElement } from 'lit-element';
+import { expect, fixture, html, nextFrame } from '@vaadin/vaadin-component-dev-dependencies/testing.js';
+import { SlottedItemsMixin } from '../../src/slotted-items-mixin';
+
+const { sinon } = window;
+
+@customElement('sim-element')
+class SimElement extends SlottedItemsMixin(LitElement) {
+  render() {
+    return html`
+      <style>
+        :host {
+          display: block;
+        }
+      </style>
+      <slot></slot>
+    `;
+  }
+}
+
+describe('SlottedItemsMixin', () => {
+  let element: SimElement;
+
+  beforeEach(async () => {
+    element = await fixture(html`
+      <sim-element>
+        <div>Item 1</div>
+        <div>Item 2</div>
+        <div>Item 3</div>
+      </sim-element>
+    `);
```

Comment:
I think it would be good to have at least some test case also for testing what happens when there are nested DOM elements and not just one level of children (e.g. a span within one of the divs). This is related to my other comment about the behaviour of `filterItems()` in the mixin file.

Reply:
Thanks, added nested elements.